task4/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <title>task-4</title>
+</head>
+<body id="bg" class="background">
+<div class="column">
+
+    <!-- LEVIY RYADOK -->
+
+    <div class="left-column">
+        <div class="box-main-1">
+            <div class="box-1">
```

Comment:
get rid of such class names, it is not reusable. Use more declarative names, e.g. 'purple-box', 'border-container'